site/src/components/theme.ts

```diff
@@ -320,6 +313,17 @@ export const theme = createTheme({
               scroll-behavior: smooth;
             }
 
+            body {
+              overflow: overlay;
```

Comment:
overlay isn't a valid value for overflow.

Reply:
It is for some browsers (see https://caniuse.com/css-overflow-overlay), when it isn't supported the browser would default to `overflow: auto`.

The reason I added this is because this prevents the Navbar from shifting slightly to the right when the login modal is opened.

Before:
https://user-images.githubusercontent.com/42802102/149164575-7d19a6e0-02a0-4619-97a0-48a6cceabb2a.mov


After:
https://user-images.githubusercontent.com/42802102/149164609-1e15b2ec-9918-47e0-acae-2aebe47c7b00.mov

But I'm not sure if this is the best solution, so I'll add a todo here for now.